src/Elmish.WPF/DynamicViewModel.fs

```diff
@@ -100,21 +157,18 @@ type [<AllowNullLiteral>] internal DynamicViewModel<'model, 'msg>
     (bindingDict    :> IReadOnlyDictionary<_,_>,
      validationDict :> IReadOnlyDictionary<_,_>)
 
-     
+  let mutable helper =
+    ViewModelHelper.create
+      (fun () -> this)
```

Comment:
Why is this a function?

Reply:
I think it was originally to avoid using references to `this` in the initialization section, but I'm not sure anymore.